qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -81,38 +81,47 @@
                 git_init_cmd = '"c:\Program Files\Git\bin\git.exe" init %s\test'
                 cdrom_cd1= "/home/winutil.iso"
                 git_check_cmd = '"c:\Program Files\Git\bin\git.exe" init %TEMP%\test'
+        - create_dir_winapi:
+            only Windows
+            only default.default.with_cache.auto.default
+            no Host_RHEL.m8
+            fs_binary_extra_options += " --log-level debug"
+            create_dir_winapi_cmd = python2.7 -c "import ctypes; dll=ctypes.windll.kernel32;dll.CreateDirectoryA('%s:\\test_winapi', 0)"
+            vfsd_winapi_log_name = "/tmp/virtiofs_create_dir_winapi_log"
+            vfsd_winapi_log_cmd = "tail -f %s > ${vfsd_log_winapi}"
```

Comment:
It seems that we did not define `vfsd_log_winapi` before referencing it.

Reply:
Yes, I modified vfsd_log_winapi's name but forgot update here.
Thanks for reminder.